policy/README.md

```diff
@@ -0,0 +1,353 @@
+
+Policies should be stored and accessible indefinitely so that the set of active policies at a given time in the past can be retrieved from the `/policies` endpoint.
+
+Policies will typically be linked to one or more associated geographies.  Geography descriptions (e.g. geofences or lists of street segments) must also be maintained by the Agency indefinitely.  Policies without specific geographies (global policies) are assumed to apply to the entire service area managed by the Agency.
+
+Policies should be re-fetched whenever (a) a Policy expires (via its `end_date`), or (b) at an interval specified by the Agency, e.g. "daily at midnight".
+
+
+<a name="schema"></a>
+## Schema
+
+<a name="policy-fields"></a>
+### Policy Fields
+
+| Name            | Type      | R/O | Description |
+| ---             | ---       | --- | --- |
+| `name`          | String    | R   | Name of policy |
+| `policy_id`     | UUID      | R   | Unique ID of policy |
+| `description`   | String    | R   | Description of policy |
+| `start_date`    | timestamp | R   | Beginning date/time of policy enforcement |
+| `end_date`      | timestamp | O   | End date/time of policy enforcement |
+| `prev_policies` | UUID[]    | O   | Unique IDs of prior policies replaced by this one |
+| `rules`         | Rule[]    | R   | List of applicable rule elements (see [“rule fields”](#rule-fields)) |
+
+Note that the provider-scope is at present not included.  A Provider should not be able to tell whether the policy is specific to them, or to a subset of Provider, or all Providers.
+
+<a name="rule-types"></a>
+### Rule Types 
+
+| Name    | Description |
+| ---     | --- |
+| `count` | Fleet counts based on regions.  Rule max/min refers to number of devices. |
+| `time`  | Individual limitations on time spent in one or more vehicle-states.  Rule max/min refers to number of minutes. |
+| `speed` | Global or local speed limits.  Rule max/min refers to miles per hour. |
+| `user`  | Information for users, e.g. about helmet laws.  Generally can't be enforced via events and telemetry. |
+
+<a name="rule-units"></a>
+### Rule Units 
+
+| Name      | Description |
+| ---       | --- |
+| `seconds` | Seconds |
+| `minutes` | Minutes |
+| `hours`   | Hours |
+| `mph`     | Miles per hour |
+| `kph`     | Kilometers per hour |
+
+<a name="rule-fields"></a>
+### Rule Fields 
+
+| Name         | Type     | R/O | Description      |
+| ---          | ---      | --- | ---              |
+| `name`       | String   | R   | Name of cap item |
+| `rule_type`  | enum     | R   | Type of policy (see [“rule types”](#rule-types))|
+| `rule_units` | enum     | O   | Measured units of policy (see [“rule units”](#rule-units))|
+| `geographies`| UUID[]   | R   | List of Geography UUIDs (non-overlapping) specifying the covered geography |
```

Comment:
I totally get your point.  However, we have a design goal of publishing immutable Policy _and_ immutable Geography objects, so that any point in time in the past can be accurately reconstructed for adjudication of Policy, and our reference implementation enforces that.  Pointers to external data sources open the door to security and cross-site issues.  I could imagine relaxing this in a subsequent revision, after input from a variety of cities interested in using this Policy / Geography paradigm.

Reply:
@rf- See update to include flat-file format